packages/devtools_app/lib/src/shared/config_specific/import_export/_export_web.dart

```diff
@@ -15,20 +16,37 @@ ExportControllerWeb createExportController() {
 class ExportControllerWeb extends ExportController {
   ExportControllerWeb() : super.impl();
 
-  @override
-  void saveFile({
-    required String content,
+  void _downloadJsObject({
+    // ignore: avoid-dynamic, there is not better way in this case
+    required dynamic content,
     required String fileName,
   }) {
     final element = document.createElement('a') as HTMLAnchorElement;
     element.setAttribute(
       'href',
-      URL.createObjectURL(Blob([content.toJS].toJS) as JSObject),
+      // ignore: avoid-dynamic, avoid_dynamic_calls, there is not better way in this case,
+      URL.createObjectURL(Blob(([content.toJS] as dynamic).toJS) as JSObject),
```

Comment:
@srujzs is this what you would expect we'd need to do here?

Reply:
I don't think this will work as intended. `toJS` doesn't exist dynamically, and if it does, it won't be consistent across all web compilers.

Does `content` need to be `dynamic`? JS interop requires that we know the type in order to convert it. If it's a set of types, we can do type checks to convert each case.

Reply:
For example if `content` is either a `String` or a `Uint8List`, we can do the following:

```dart
JSAny jsContent = content is String ? content.toJS : (content as Uint8List).toJS;
URL.createObjectURL(Blob([content].toJS]) as JSObject);
```

Reply:
Thanks. This way .toJS does not resolve for Blob. So, I made it little differently.

Reply:
Looks good to me!